app/screens/settings/notification_call/notification_call.tsx

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useMemo, useState} from 'react';
+import {useIntl} from 'react-intl';
+import InCallManager from 'react-native-incall-manager';
+
+import {updateMe} from '@actions/remote/user';
+import SettingBlock from '@components/settings/block';
+import SettingContainer from '@components/settings/container';
+import SettingOption from '@components/settings/option';
+import SettingSeparator from '@components/settings/separator';
+import {Calls} from '@constants';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import useAndroidHardwareBackHandler from '@hooks/android_back_handler';
+import useBackNavigation from '@hooks/navigate_back';
+import {t} from '@i18n';
+import {popTopScreen} from '@screens/navigation';
+import {changeOpacity} from '@utils/theme';
+import {getNotificationProps} from '@utils/user';
+
+import type UserModel from '@typings/database/models/servers/user';
+import type {AvailableScreens} from '@typings/screens/navigation';
+
+type Props = {
+    componentId: AvailableScreens;
+    currentUser?: UserModel;
+};
+
+const footerText = {
+    id: t('notification_settings.calls.callsInfo'),
+    defaultMessage: 'Note: silent mode must be off to hear the ringtone preview.',
+};
```

Comment:
I think we can use `defineMessages` from react-intl instead. there are some examples in the code, just search for `defineMessages`

Reply:
Ah, thanks!